### ISSUE TITLE
Support javax.validation.ValidationException

### ISSUE THREAD

**Issue by SchulteMarkus** [cloudstark/quarkus-problem-extension#2]
Imagine a method

```java
@Path("/calc")
public class CalcResource {
 ...
  @GET
  @Path("/validation")
  @Produces(TEXT_PLAIN)
  public String violation(@QueryParam("name") @NotBlank final String name) {
    return name;
  }
}
```

Using this quarkus-problem-extension, there is still not a "problem"  returned when calling this method:

```bash
$ curl -v http://localhost:8080/calc/validation?name=
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8080 (#0)
> GET /calc/validation?name= HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 400 Bad Request
< Content-Length: 56
< validation-exception: true
< Content-Type: text/plain;charset=UTF-8
```

**Feature request**: Please represent validation errors as a "problem", too.

**Some thoughts**
- javax.validation.ValidationException is thrown in case of validation error
- This Quarkus extension is trying to catch every Exception in [RestExceptionMapper](https://github.com/cloudstark/quarkus-problem-extension/blob/master/runtime/src/main/java/solutions/cloudstark/quarkus/problem/runtime/RestExceptionMapper.java#L31). This seems not to work
- Probably because there is already [ResteasyViolationExceptionMapper](https://github.com/quarkusio/quarkus/blob/master/extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/jaxrs/ResteasyViolationExceptionMapper.java#L22) present in quarkus-hibernate-validator which takes care for ValidationExceptions

**Comment by SchulteMarkus** [comment]
I don't know an answer to the question: How to replace [ResteasyViolationExceptionMapper](https://github.com/quarkusio/quarkus/blob/master/extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/jaxrs/ResteasyViolationExceptionMapper.java#L22) as part of this extension? And furthermore, a Quarkus user using quarkus-problem-extension does not have to be using quarkus-hibernate-validator.

**Comment by CrystalMethod** [comment]
Hi @SchulteMarkus. Thanks for your feedback and thoughts about this extension. Especially since I am still learning new Quarkus internals every day. Yes, we are using this `quarkus-probleme-extension` in conjunction with the Hibernate Validator. In this case we re using "service method validation" [1].

But anyway: We could try to play with `@Priority`.

[1] https://quarkus.io/guides/validation#rest-end-point-validation

**Comment by SchulteMarkus** [comment]
@CrystalMethod Until now I am only using Quarkus, so I don't know internals, too. But you might be right regarding ```@Priority```, as used in Quarkus itself, for example at [ForbiddenExceptionMapper](https://github.com/quarkusio/quarkus/blob/master/extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/ForbiddenExceptionMapper.java#L15).

**Comment by CrystalMethod** [comment]
fixed with 1cd78af1519f7082b411d8ed862c3cbb0a78bb7d
